cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/GoalOptimizer.java

```diff
@@ -250,7 +189,7 @@ private int numProposalComputingThreads() {
     return _numPrecomputingThreads > 0 ? _numPrecomputingThreads : 2;
   }
 
-  private void computeBestProposal() {
+  private void computeCachedProposal() {
     long start = _time.milliseconds();
     List<Future> futures = new ArrayList<>();
     for (int i = 0; i < (_numPrecomputingThreads > 0 ? _numPrecomputingThreads : 1); i++) {
```

Comment:
maybe we can remove this for loop since `(_numPrecomputingThreads > 0 ? _numPrecomputingThreads : 1)` is always 1 given the new config constraint we put on `NUM_PROPOSAL_PRECOMPUTE_THREADS_CONFIG`